HannaPidchasiuk/HW7/task4.py

```diff
@@ -0,0 +1,22 @@
+# You have a file of unknown length.
+# Write a function that will remove all numbers from each line of the file.
+import re
+
+
+def remove_numbers(lines: list) -> list:
```

Comment:
it is better to have file_path as a parameter, and read file inside the function